public/js/business_add_review.js

```diff
@@ -0,0 +1,77 @@
+const sendButton = getElement('send-button');
+const textReview = getElement('text-review');
+// const evaluation = getElement('evaluation');
+const currentData = getElement('current-data');
+const nameBusiness = getElement('name-business');
+
+const stars = document.querySelectorAll('.stars li');
+let eval = 0;
```

Comment:
eval is a global function that evaluates string values. You can't reassign in in ES6. This is a fatal error.

Pay attention to eslint.